docs/platform.rst

```diff
@@ -1,22 +1,47 @@
+.. _platform:
+
 Platform
 ========
 
-contains platform details
+OpenQL supports various target platforms. These platforms can be software simulators or architectures targetting hardware quantum computers. Following platforms are supported by OpenQL:
+
+- software simulator platforms
+	- `QX <https://github.com/QE-Lab/qx-simulator>`_
+	- `Quantumsim <https://gitlab.com/quantumsim/quantumsim>`_
+- hardware platforms
+	- CC-Light
+	- CC
+	- CBOX
+
+
+Platform can be created in OpenQL by using ``Platform()`` API as show below:
+
+.. code:: python
+
+    platform = ql.Platform('<platform_name>', <path_to_json_config_file>)
+
+For example, a platform with the name ``CCL_platform`` and using ``hardware_config_cc_light.json`` can be created as:
 
-platform is specified in configuration file
+.. code:: python
 
+    platform = ql.Platform('CCL_platform', 'hardware_config_cc_light.json')
 
-Generic fields of configuration file
-------------------------------------
 
+Platform Configuration File
```

Comment:
What does this configuration file affect? Specify precisely whether this configuration affects only the OpenQL backends, or if the configurations provided in the Platform can have any effects on scheduling, mapping, etc 

Basically, I believe a better explanation of what role platform constraints play in the compilation chain is needed.

Reply:
the role played by each of these constraints will be elaborated in respective sections. here it is only mentioned the configuartion, now how these contraints are used, for instance in scheduling, and how is output affected by these constraints will be discussed in scheduler